src/crimson/osd/pg.cc

```diff
@@ -907,15 +907,28 @@ void PG::mutate_object(
   }
 }
 
+void PG::enqueue_push_for_backfill(
+  const hobject_t &obj,
+  const eversion_t &v,
+  const std::vector<pg_shard_t> &peers)
+{
+  assert(recovery_handler);
+  assert(recovery_handler->backfill_state);
+  auto backfill_state = recovery_handler->backfill_state.get();
+  backfill_state->enqueue_standalone_push(obj, v, peers);
```

Comment:
PGBackend should probably have a reference to backfill_state/recovery_handler.  Rather than enqueue_standalone_push, how about notify_object_created()?  I'd prefer that PG/PGBackend be notifying the backend of an object being created rather than specifying that a push needs to happen.